### DISCUSSION TITLE
Alerts & animations

### DISCUSSION THREAD

**Discussion by johannaweintraub** [twilio-labs/paste#690] [Q&A] [answered]
Is there a simple way to add animation to an alert that is pushing content down?

**Comment by TheSisb** [comment, marked as answer]
Hey @johannaweintraub,

What kind of animation are you looking to do? For example, is the alert sliding-in and you also want the neighboring content to slide down as well? If so, you might be able to achieve a slide-in effect with CSS.  

For more complex animations, we recommend using our `@twilio-paste/animation-library`package, which uses [react-spring](https://www.react-spring.io/) under the hood. It's still in alpha status, meaning our documentation and guidance is still a work in progress. This also means we're more willing to help collaborate and use this experience to help shape the animation system. :)  

If you're able to, please share some more context on what you're trying to achieve and if it's too much for a thread here, we can even set some time to work the problem in a DSYS Office Hour.

**Comment by johannaweintraub** [comment]
Thanks so much for the quick response. Yes, that is the animation I want to add. I will discuss with my engineers and come to office hours if there is more to discuss.

**Reply by jweintraub** [reply]
Following up on this old thread - is there any guidance on animations, optimal speeds, etc? We're planning to show an animation in the nav when we go in and out of a service nav.

**Reply by SiTaggart** [reply]
Do you mind creating a new discussion topic for general animation guidelines for discoverability please, @johannaweintraub ?

**Reply by jweintraub** [reply]
done https://github.com/twilio-labs/paste/discussions/1660
